pkg/cloud/services/ssm/secret_test.go

```diff
@@ -39,6 +42,31 @@ import (
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
+type mockAPIError struct {
+	Code    string
+	Message string
+}
+
+// ErrorCode returns the error's code, making it satisfy one part of the interface.
+func (e *mockAPIError) ErrorCode() string {
+	return e.Code
+}
+
+// ErrorMessage returns the error's message.
+func (e *mockAPIError) ErrorMessage() string {
+	return e.Message
+}
+
+// Error returns the error's message, satisfying the standard 'error' interface.
+func (e *mockAPIError) Error() string {
+	return e.Message
+}
```

Comment:
Oh just realized, don't these do the same thing?

Reply:
I have made the commit to make returns consistent in tests. Though these were not same , but your review made me think on the errors I used in returned for tests , they look good to me now.